modules/agent-policy/scripts/script-utils.sh

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+####################################################################
+## This script contains util functions. The util functions are used
+## in modules/agent-policy/scripts/create-update-script.sh and
+## modules/agent-policy/scripts/delete-script.sh
+####################################################################
+
+
+CREATE="create"
+UPDATE="update"
+LAUNCH_STAGE="alpha"
+
+
+# Params:
+#   $1 = JSON formatted list(string)
+# Return:
+#   A well-formatted command line flag value for a list of strings
+function get_formatted_list_of_strings() {
+    local formatted
+    local python="python -c 'import json, sys;"
```

Comment:
We should avoid depending on python, especially if it's just for basic formatting. Use jq instead.

Reply:
I thought it was okay to use python because python is required in order to run the gcloud commands? The documentation for that is [here](https://cloud.google.com/sdk/docs?hl=en).

Reply:
Yes, but it's still an external dependency we'd rather avoid.

Reply:
That's a good point, but I feel like the string manipulation in these couple methods is not that basic, especially `get_formatted_list_of_objects` and `get_formatted_list_of_list_of_objects`.

Reply:
It's still pretty basic and could be done with `jq`, but we can track this later via #4.